ushahidi/platform-client
# Making edit-button for saved-searches reachable from keyboard

This pull request makes the following changes:
- Makes the edit-button beside the saved-searches dropdown in the filter-dropdown reachable from the keyboard

Testing checklist:
- Make sure you are logged in as an admin
- Open the filters-dropdown
- Select a saved search from the dropdown
- Try tabbing around in the filter-dropdown after this
- [ ] You should reach the edit-button that's positioned after the saved-search-dropdown from the keyboard


Fixes ushahidi/platform#4195 .

Ping @ushahidi/platform
